prov/hook/src/hook_domain.c

```diff
@@ -101,6 +102,82 @@ static struct fi_ops_mr hook_mr_ops = {
 	.regattr = hook_mr_regattr,
 };
 
+static ssize_t hook_credit_handler(struct fid_ep *ep_fid, size_t credits)
+{
+	/* called from the base provider, ep_fid is not hook ep */
```

Comment:
nit: Can we say what ep_fid is, rather than what it is not?

Reply:
Fixed.